store/accessCodes.ts

```diff
@@ -17,7 +18,7 @@ export const useAccessCodesStore = defineStore('accessCodes', () => {
     }
 
     const verifyAccessCode = (request: AccessCode) => {
-        const req = store.$nuxt.$apiMulti.accessCodes.verifyAccessCode(request)
+        const req = apiMulti.value.accessCodes.verifyAccessCode(request)
```

Comment:
Was the typing improved with Nuxt 3 ?

Reply:
Yes, much improved. All exposed to the nuxt instance `useNuxtApp()`